src/hammer-tech/specialcells.py

```diff
@@ -60,7 +62,7 @@ class SpecialCell(NamedTuple('SpecialCell', [
     def from_setting(d: dict) -> "SpecialCell":
         size = d['size']
         if size is not None:
-            size = Decimal(str(d['size']))
+            size = list(map(lambda x: str(x), size))
```

Comment:
Do we want to add a check here to ensure that the length of the size field matches the length of the name field?

Reply:
The check is done in the plugin, but does it make sense to do it here instead, or do you think it's better to just retain flexibility for using the size field with any of the special cells?  I suppose it doesn't matter much either way.

Reply:
I think it's better to retain flexibility for now, since "size" is only being used by one special cell at the moment, and future use cases of this field are TBD.